packages/react-microlink/src/components/Card/CardEmptyState.js

```diff
@@ -58,15 +66,18 @@ const EmptyLink = styled.span`
   animation-delay: .25s;
 `
 
-const CardEmptyState = ({cardSize}) => (
-  <Fragment>
-    <EmptyImage cardSize={cardSize} />
-    <Content cardSize={cardSize}>
-      <EmptyTitle />
-      <EmptyDescription />
-      <EmptyLink />
-    </Content>
-  </Fragment>
-)
+const CardEmptyState = ({cardSize}) => {
+  console.log('cardSize', cardSize)
```

Comment:
You've got a console.log in here @Kikobeats, also ideally it should be a _direct_ return from the arrow function, ie.

```jsx
const CardEmptyState = ({cardSize}) => (
  <Fragment/>
);
```

Reply:
😱 thanks, removed